lib/fog/core/service.rb

```diff
@@ -163,6 +163,8 @@ def coerce_options(options)
           value_string = value.to_s.downcase
           if value.nil?
             options.delete(key)
+         elsif key.end_with? 'name'
```

Comment:
Ins't `end_with` available only on ruby `2.6+`?

Reply:
Seems not (checked ✅on 2.3) https://apidock.com/ruby/String/end_with%3f